.github/actions/check-merge-conflict/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Check For Merge Conflict
+description: Checks for merge conflict in Pull Request
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check for dirty PR
+      uses: eps1lon/actions-label-merge-conflict@b8bf8341285ec9a4567d4318ba474fee998a6919
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=For%20readability%2C%20also%20comment%20the%20version%20there.